services/orchest-webserver/client/src/components/MultiSelect.jsx

```diff
@@ -26,6 +26,7 @@ const isNumeric = (value) => value.match("^\\d+$") != null;
  *    required?: boolean;
  *    type?: "text" | "number";
  *    value: TMultiSelectInputValue;
+ *    onBlur: (event: any) => void
  *  } &
  *  Record<
  *    "onChange" | "onKeyDown" | "onKeyUp",
```

Comment:
```suggestion
 *    "onBlur" | "onChange" | "onKeyDown" | "onKeyUp",
```

This will add the appropriate input event type (rather than `any`)

Reply:
This doesn't work as onBlur doesn't qualify as a keyboard event and 

``` * @typedef { React.KeyboardEvent<HTMLInputElement>
 *  & { target: HTMLInputElement }
 * } TMultiSelectInputEvent```

Reply:
In that case let's add the relevant type rather than `any`